concourse/pipeline.yml

```diff
@@ -1,63 +1,55 @@
-##########################################################
-#
-# Groups are used to display jobs on the Web UI
-#
-##########################################################
-groups:
-- name: sagan
-  jobs:
-   - build-compass
+resource_types:
+- name: cf-zero-downtime-resource
+  type: docker-image
+  source:
+    repository: emeraldsquad/cf-zero-downtime-resource
+    tag: latest
+- name: slack-notification
+  type: docker-image
+  source:
+    repository: cfcommunity/slack-notification-resource
+    tag: latest
 
-##########################################################
-#
-# Resources a specific instances of resources to be used
-# by jobs : a Git repo, a PCF instance, etc.
-#
-##########################################################
 resources:
-- name: sagan-repo
+- name: sagan-site-repo
   type: git
   source:
-    uri: ((sagan-git-uri))
-    branch: ((sagan-git-branch))
-- name: sagan-cf-qa
-  type: cf
+    uri: "https://github.com/spring-io/sagan.git"
+- name: sagan-cf-production
+  type: cf-zero-downtime-resource
   source:
     api: https://api.run.pivotal.io
-    username: ((compass-cf-login))
-    password: ((compass-cf-password))
-    organization: pivotallabs
-    space: Beach-Paris
-    skip_cert_check: false
-
-##########################################################
-#
-# Jobs are tasks executed by concourse.
-#
-##########################################################
+    username: ((sagan-cf-login))
+    password: ((sagan-cf-password))
+    organization: spring.io
+    space: production
+- name: slack-alert
+  type: slack-notification
+  source:
+    url: ((slack-webhook-url))
 
 jobs:
-- name: build-compass
+- name: deploy-sagan-site
   serial: true
   plan:
-  - get: sagan-repo
+  - get: sagan-site-repo
     trigger: true
     params: { submodules: none }
-  - task: build-sagan
+  - task: build-sagan-site
     params:
       # JSPM github authentication is needed for the CI
       # otherwise install steps reach github API limitation
-      JSPM_GITHUB_AUTH_TOKEN: ((JSPM_GITHUB_AUTH_TOKEN))
+      JSPM_GITHUB_AUTH_TOKEN: ((jspm-github-auth-token))
     config:
```

Comment:
I prefer splitting this out into it's own `.yml` file for the task because it's easier to read.